server/athenian/api/serialization.py

```diff
@@ -193,6 +193,11 @@ def _deserialize_dict(data: dict, boxed_type, path: str) -> dict:
     return {k: _deserialize(v, boxed_type, f"{path}.{k}") for k, v in data.items()}
 
 
+def serialize_date(d: date) -> str:
+    """Serialize a date object into a string."""
+    return d.isoformat()
```

Comment:
Which is the same as `"%Y-%m-%d"`, right? No US gotchas?

Reply:
yes it's the same https://github.com/python/cpython/blob/3.8/Modules/_datetimemodule.c#L3162